create_stats/boxplots.py

```diff
@@ -16,25 +16,29 @@ def box_for_spec(grades):
     ax.boxplot(grades, labels = ["MATE-INFO", "FILO", "STIINTE"])
     ax.set_ylabel("NOTE")
     ax.set_xlabel("SPECIALIZARE")
-    ax.set_title("BOXPLOT")
+    ax.set_title("BAC 2018 specialisation comparison")
     plt.show()
 
+def return_grades_as_array(l1, l2, l3):
+        mate, filo, stiinte = [], [], []
+        for x in l1:
+            mate.append(x.final_grade)
+        for x in l2:
+            filo.append(x.final_grade)
+        for x in l3:
+            stiinte.append(x.final_grade)
+        return mate, filo, stiinte
+
```

Comment:
this should only get one list and return one array because we don't know how many specialisations there will be in the future